postgres/assets/configuration/spec.yaml

```diff
@@ -359,10 +359,36 @@ files:
         - name: collection_interval
           description: |
             Set the query sample collection interval (in seconds). Each collection involves a single query to
-            `pg_stat_activity` followed by at most one `EXPLAIN` query per unique normalized query seen.
+            `pg_stat_activity` followed by at most one `EXPLAIN` query per unique normalized query seen if 
```

Comment:
I don't think you need to change this description, since `query_activity` has its own config with a collection interval that the user can change

Reply:
there is an argument to made to just change `query_samples` to `explain_plans` though

Reply:
Good point for the first part. I had modified this just to be extra sure that people understand that if they disable explain plan collection there won't be explain plans ^^. However, the wording is already "at most" so we could probably get away with not changing this.

Reply:
For the second one, I'm not 100% sure I follow. Are you talking about the `query_samples` parameter in the spec/config file?  If so, I think it's better to keep both because (in the support case that started this) the user wanted to collect the samples along with activity metrics. They just didn't want the explains because they're using extended query protocol and the explain parameterized queries function requires them to open permissions for their entire database.  They don't find the explains that helpful and would rather just have the samples and activity metrics.

No matter what though, I think that would probably need a second PR if we're going to change that as well no?